Bottom: 326c207fec6c76aba30460a024eb9d8aee4de7f9
Top:    d5b0bb3355b59dd818b25eaeb81a1e6e5e55e0b2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 17:20:28 -0500

Refresh of jds-lirc-gpt

---

diff --git a/drivers/input/ir/ir-gpt.c b/drivers/input/ir/ir-gpt.c
index 16e1d5a..ab83d89 100644
--- a/drivers/input/ir/ir-gpt.c
+++ b/drivers/input/ir/ir-gpt.c
@@ -125,7 +125,7 @@ static int __devinit ir_gpt_of_probe(struct of_device *op,
 	ret = input_register_device(ir_gpt->input);
 	if (ret)
 		goto free_input;
-	ret = input_ir_register(ir->input);
+	ret = input_ir_register(ir_gpt->input);
 	if (ret)
 		goto free_input;
